hack/clean.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# **clean.sh**
+
+cd $GOPATH/src/github.com/kubeedge/kubeedge
+
+removefolder() {
+if [ -d bin ]; then
+rm -rf bin
+fi
+
+if [ -d .idea ]; then
+rm -rf .idea
+fi
+}
+
+#terminate mosquitto if running
+killmosquitto() {
+if  pgrep mosquitto  >/dev/null 2>&1  ; then
+     pkill -9 mosquitto
+fi
+
+}
+
+#terminate edge core if running
+killedge () {
+if pgrep edge_core >/dev/null 2>&1 ; then
+     pkill -9 edge_core
+fi
+
+}
+
+#delete all logs
+cleanfiles () {
+
+if [ -f edge_core ]; then
+rm -rf edge_core
+fi
+find . -type f -name "*db" -exec rm -f {} \;
+find . -type f -name "*log" -exec rm -f {} \;
+find . -type f -name "*out" -exec rm -f {} \;
+}
+
+removefolder
+killmosquitto
```

Comment:
You don't have to kill MQTT. There is no need as there is no development activity with in MQTT or any configurations which we are changing during our development, which requires it to be restarted everytime.

Reply:
@rohitsardesai83 -is that ok not to kill MQTT broker , while after calling make clean?

Reply:
i feel its fine not to kill MQTT broker.

Reply:
ok